src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -27,125 +27,249 @@
-                    #formsubmit></app-input>
+      <div *ngIf="challenge.is_docker_based" class="ev-md-container
+        ev-card-panel">
+        <span class="fs-20 fw-light">
+          Submission limits
+        </span><br><br>
+        <span class="fs-15">
+          <strong class="fw-semibold content">Team name:</strong>
+          {{phaseRemainingSubmissions.participant_team}}
+        </span>
+        <table align="center" class="centered responsive-table">
+          <thead>
+            <tr>
+              <th class="content">Phase details</th>
+              <th class="content">Remaining submissions</th>
+              <th class="content">Submission command</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr *ngFor="let key of phaseRemainingSubmissions.phases">
+              <td class="content">{{key.name}}
+                <br>
+                <strong class="text-med-black">Start date: </strong>{{key.start_date | date:'medium'}}
+                <br>
+                <strong class="text-med-black">End date: </strong>{{key.end_date | date:'medium'}}
+              </td>
+              <td *ngIf="phaseRemainingSubmissionsFlags[key.id] === 'showSubmissionDetails'"
+              class="content">
+                <strong class="fw-semibold">
+                  Today:
+                </strong>
+                {{key.limits.remaining_submissions_today_count}}<br>
+                <strong class="fw-semibold">
+                  This month:
+                </strong>
+                {{key.limits.remaining_submissions_this_month_count}}<br>
+                <strong class="fw-semibold">
+                  Total:
+                </strong>
+                {{key.limits.remaining_submissions_count}}
+              </td>
+              <td *ngIf="phaseRemainingSubmissionsFlags[key.id] === 'showClock'" class="content">
+                <strong class="fw-semibold">{{key.limits.message}}</strong><br>
+                Next submission will be available in<br>
+                <div>
+                  <span
+                    *ngIf="days!=0">{{phaseRemainingSubmissionsCountdown[key.id].days}}
+                    Days, </span>{{phaseRemainingSubmissionsCountdown[key.id].hours}}
+                  hours : {{phaseRemainingSubmissionsCountdown[key.id].minutes}}
+                  minutes :
+                  {{phaseRemainingSubmissionsCountdown[key.id].seconds}} seconds
                 </div>
-              </div>
+              </td>
+              <td *ngIf="phaseRemainingSubmissionsFlags[key.id] === 'maxExceeded'"
+              class="content">
+                <strong>{{key.limits.message}}</strong><br>
+                <strong>
+                  {{key.limits.maxExceededMessage}}
+                </strong>
+              </td>
+              <td>
+                <code>$ evalai push &lt;image&gt;:&lt;tag&gt; --phase {{key.slug}}</code>
+                <button class="btn ev-btn-dark btn-waves-effect waves-dark
```

Comment:
When I checked it, `key.slug` was NULL. So, nothing was printed after `--phase` .

Reply:
done